docs/workflows.md

```diff
@@ -0,0 +1,65 @@
+# Workflows for some common use cases 
+
+## 1.- Add a new feature at the end of the feature list.
+
+Manual actions to do by the dev:
+
+* Create a new branch with the feature using the last feature as base.  
+* Create a new PR with the new branch to get the review.  
+
+Manual actions to do by the review:
```

Comment:
Reviewer not review

Reply:
Done here https://github.com/fluendo/git-upstream-workflow/commit/ab28d87f116006f0ec7db41291a58a1cffce39f5